velodyne_lidar_driver_wrapper/launch/velodyne_lidar_driver_wrapper_launch.py

```diff
@@ -58,27 +59,27 @@ def generate_launch_description():
     declare_gps_time = DeclareLaunchArgument(name = 'gps_time', default_value = 'False')
 
     # Args for Pointcloud
-    velodyne_pointcloud_pkg = get_package_share_directory('velodyne_pointcloud')
-
-    organize_cloud = LaunchConfiguration('organize_cloud')
-    declare_organize_cloud = DeclareLaunchArgument(name ='organize_cloud', default_value = 'False')
```

Comment:
It looks like the previous version of this `velodyne_lidar_driver_wrapper_launch.py` launch file already attempted to launch `velodyne_point_cloud/launch/velodyne_conver_node-VLP32C-launch.py` with `organize_cloud` set to `False`. Did you find that the original version didn't result in `organize_cloud` being set to `False`?

Reply:
Yeah I found that setting `organize_cloud` didn't have an impact on the driver because this variable is not declared as a launch argument in `velodyne_convert_node-VLP32C-launch.py`. Instead, it is read in from `VLP32C-velodyne_convert_node-params.yaml`. My solution was to migrate this config and the logic present in `velodyne_convert_node-VLP32C-launch.py` to the wrapper.

Reply:
Okay thanks for confirming! Just wanted to check that the original implementation did not work properly. Would you mind updating the Pull Request description to mention that this fix was needed?